setup.py

```diff
@@ -103,6 +103,7 @@
     'Source': 'https://github.com/secure-systems-lab/securesystemslib',
     'Issues': 'https://github.com/secure-systems-lab/securesystemslib/issues',
   },
+  python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, !=3.5.*, <4",
```

Comment:
So this means that any users of python 3.5 will now be offered securesystemslib 0.18 by the pypi clients even if new securesystemslib versions are released, correct?

I guess this is correct and reasonable (since cryptography does exactly that), just wanted it to be explicit.

Reply:
> So this means that any users of python 3.5 will now be offered securesystemslib 0.18 by the pypi clients even if new securesystemslib versions are released, correct?

Good question. The way I read [the docs](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires) this makes sure that pip on e.g. 3.5 will not install the next version of securesystemslib, i.e. > 0.18. Not sure if this also means that it will fall back to the latest version, i.e. 0.18, or just refuse to install. I could live with both.

Btw. I didn't add this line because I dropped 3.5, but because I saw that it was missing while dropping 3.5, in case this makes a difference. :)